Cpp/fostgres-fg/fg.builtin.http.cpp

```diff
@@ -47,8 +47,10 @@ namespace {
             auto response = fg::mime_from_argument(
                     stack, stack.argument("response", pos, end));
             fg::assert_comparable(*actual.first, *response);
+            return fostlib::json();
         }
-        return fostlib::json();
+        return body_data(*actual.first);
```

Comment:
I kind of feel this should be in an `else`... But not very strongly.

Also, I think it's important that there's at least a comment here to describe the implication of this logic. That is, that the body data is only returned if the test didn't check the outcome of the request.